deploy/build/README.CentOS.bash

```diff
@@ -31,7 +31,7 @@ sudo yum install -y \
     postgresql-devel
 
 sudo pip3 install conan
-sudo pip3 install -r python-dependencies.txt
+sudo pip3 install -r $GITHUB_WORKSPACE/python-dependencies.txt
```

Comment:
Don't change this path.

Reply:
README.CentOS needs to use python-dependencies.txt during execution. Before moving python-dependencies.txt, you can get it. But now, if you do not modify the location, the script cannot get the location of python-dependencies.txt. An error will be reported

Reply:
Ok, got it. Please update the file path to the latest path. Do not use variable $GITHUB_WORKSPACE, which is not defined in the bash script.